apps/backoffice-v2/src/router.tsx

```
@@ -216,7 +220,44 @@ const router = createBrowserRouter([
+          return handler(context);
+        })
+      )
+    );
+
+    console.assert(matchesToLoad.length === results.length, 'result must match matches in length');
+
+    return results.reduce(
+      (acc, result, i) =>
+        Object.assign(acc, {
+          // Length of matches and results is asserter above
+          [matchesToLoad[i]!!.route.id]: result,
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)